src/components/country-code/index.tsx

```diff
@@ -0,0 +1,170 @@
+import styled from "styled-components";
+import React, { useState } from "react";
+import icon_dropdown from "../../assets/arrow_drop@.png";
+import array from "./info";
+import OutsideAlerter from "../OutsideClickAlert";
+const TelephonePicker = ({ isOptional }: { isOptional: any }) => {
+  const [dropDownStatus, setDropDownStatus] = useState(false);
+  const [selectedCountry, setSelectedCountry] = useState("india");
+  const [selectedCode, setSelectedCode] = useState("91");
+  const handleSelection = (e: any) => {
+    setSelectedCountry(e.currentTarget.id);
+    setSelectedCode(
+      array.find((element) => element.country === e.currentTarget.id)!.code
+    );
+    setDropDownStatus(false);
+  };
+
+  return (
+    <FormContainer>
+      {isOptional ? <Label>Mobile No</Label> : <Label>Mobile No.</Label>}
+      <MobileNumberContainer>
+        <OptionSelected>
+          <OptionWrapper>
+            <Flag src={`/assets/images/${selectedCountry}.png`}></Flag>
```

Comment:
Don't use relative path here, give reference from separte file.

Reply:
Have used variable for the path